docs/doc_example/CMakeLists.txt

```diff
@@ -84,14 +91,13 @@ add_onnx_mlir_executable(OMRuntimeTest
   OMExecutionSession
   )
 
+set_output_directory(OMRuntimeCppTest BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR} LIBRARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
-set_output_directory(OMRuntimeTest BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR} LIBRARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
-add_test(NAME OMRuntimeTest COMMAND OMRuntimeTest WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+add_test(NAME OMRuntimeTest COMMAND OMRuntimeTest WORKING_DIRECTORY ${ONNX_MLIR_BIN_ROOT})
```

Comment:
Doesn't this fix make it unnecessary to change change the test working directory back to root?

Reply:
I see you fixed this after, sorry!